### PR TITLE
Finder style and component polish

### PR DESCRIPTION
**Pull request by jordanwade**
Pretty straight forward clean up PR, that addresses some responsive issues and moves a section of duplicate code into a component to be shared on community `index.js` and `CommunitySingle.js`.

### CODE REVIEW

File: pages/community/Community.styles.js [hunk 1]
```diff
@@ -26,10 +26,9 @@ const Hero = styled.div`
 `;
 
 const Title = styled.h1`
-  font-size: 75px;
   /* Fallback: Set a background color. */
   background-color: red;
-
+  font-size: 50px;
```
Comment (drewbarontini) [inline pages/community/Community.styles.js:31]: We should probably use the `rem()` function for these…

File: pages/community/Community.styles.js [hunk 2]
```diff
@@ -49,6 +48,13 @@ const Title = styled.h1`
   -moz-background-clip: text;
   -moz-text-fill-color: transparent;
 
+  @media screen and (min-width: ${themeGet('breakpoints.md')}) {
+    font-size: 65px;
```
Comment (drewbarontini) [inline pages/community/Community.styles.js:52]: `rem('65px')`

File: pages/community/Community.styles.js [hunk 3]
```diff
@@ -49,6 +48,13 @@ const Title = styled.h1`
...
+  }
+
+  @media screen and (min-width: ${themeGet('breakpoints.lg')}) {
+    font-size: 75px;
```
Comment (drewbarontini) [inline pages/community/Community.styles.js:56]: `rem('75px')`

Comment (rkdavidson) [reply pages/community/Community.styles.js]: Fixed in commit af646d0

**Review by rkdavidson — APPROVED** [review]
Tested locally and looks good, addressed @drewbarontini 's feedback re: `rem()` helper for pixel values in commit af646d0.

**Pull request merged**